### PR TITLE
Changed a couple of things in second.md

### PR DESCRIPTION
**Pull request by oscarpicasso**
This is a minor change.

### CODE REVIEW

**Comment by oscarpica** [comment]
What are the new features introduced by this change?

**Comment by oscarpicasso** [comment]
The features are not that great. This just allow the reader to read one more line.

File: second.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # This is a second file header
+
+Changed a few things in second file.
```
Comment (oscarpica) [inline second.md]: What are the few things that changed?

File: second.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # This is a second file header
+
```
Comment (oscarpica) [inline second.md]: Not sure this blank line is needed.

Comment (oscarpica) [inline second.md]: Another comment

Comment (oscarpica) [inline second.md]: I want to start a review

File: second.md (hunk 1 above)
Comment (oscarpica) [inline second.md]: Another review. Make is more read.

File: second.md (hunk 2 above)
Comment (oscarpica) [inline second.md]: Make if more blue.

**Pull request merged**